src/status_im/subs/communities.cljs

```diff
@@ -414,7 +420,7 @@
                                            checking?
                                            token-images)
                                   tokenRequirement))
-                               (:criteria role))})
+                               (visible-criteria (:criteria role)))})
```

Comment:
We might benefit from using spec on the criteria to have an idea about it's type.